src/core/transitions/transition-utils.js

```diff
@@ -41,7 +41,7 @@ export function getEndValueByShortestPath(propName, startValue, endValue) {
 }
 
 // TODO/xiaoji: This should be merged with the controller's prop constraint system
-export function extractViewportFrom(props) {
+export function extractViewState(props) {
```

Comment:
This is WebMercatorViewport specific. Should this util be offered by the Viewport/View class?

Reply:
Yes good idea. This diff just renames the function to reduce `View/Viewport/ViewState` confusion. 

Updated #1370 to make sure we address after this lands.